apps/meteor/app/integrations/server/api/api.js

```diff
@@ -270,10 +270,11 @@ function executeIntegrationRest() {
 	this.bodyParams.bot = { i: this.integration._id };
 
 	try {
-		const message = processWebhookMessage(this.bodyParams, this.user, defaultValues);
-		if (_.isEmpty(message)) {
-			return API.v1.failure('unknown-error');
-		}
```

Comment:
Couldn't we keep this check anyways?

Reply:
you mean the channel in `bodyParams`? I see that `_.isEmpty()` would still return `true` if there's no default channel.

Reply:
re-added condition